src/qrl/core/TransactionPool.py

```diff
@@ -15,6 +15,17 @@
 
 
 class TransactionPool:
+    """
+    As each Transaction comes in over the network, it is first stored in
+    TransactionPool.pending_tx_pool before the node validates them. After the TX
```

Comment:
1. Replace TX by Transaction. Please don't use short forms while writing documentation.
2. It is worth mentioning that it keeps track of unconfirmed transaction.
Probably you should start like, TransactionPool keeps track of unconfirmed/pending transactions.
3. It is worth to talk about what it does, before you talk about how it does.

Reply:
I won't use the word 'unconfirmed' here because it also implies a Transaction that has been mined into a Block, but the Block isn't old enough. I'll figure out some other words...

Reply:
A transaction which has been mined, are removed as soon as block has been added to the chain.